src/status_im/contexts/profile/backup_recovery_phrase/style.cljs

```diff
@@ -53,8 +54,28 @@
    :left     0
    :right    0})
 
-(defn description-text
-  [theme]
-  {:margin-horizontal 40
-   :text-align        :center
-   :color             (colors/theme-colors colors/neutral-80-opa-70 colors/white-opa-70 theme)})
+(def container {:flex 1})
+
+(def seed-phrase {:margin-horizontal 20})
+
+(def words-colum
+  {:flex             1
+   :row-gap          12
+   :padding-vertical 8})
+
+(defn seed-phrase-blur-overlay
+  [shell? theme]
+  (cond-> {:flex 1}
+    (and platform/android? shell?)   (assoc :background-color "#3f4959")
```

Comment:
this looks like some magic, why do we need all these ? and custom color not from colors ns

Reply:
I forgot to add the comment 🤦‍♂️

This is the overlay opaque color calculated for Android, since we don't have blur. I calculate it by using an online calculator to add colors, but I'll check if a color already defined fits well.

Thanks for spotting it!